src/ucp/core/ucp_context.c

```diff
@@ -946,14 +945,15 @@ static int ucp_tl_resource_is_same_device(const uct_tl_resource_desc_t *resource
            (resource1->sys_device == resource2->sys_device));
 }
 
-static void ucp_add_tl_resource_if_enabled(ucp_context_h context, ucp_tl_md_t *md,
-                                           ucp_md_index_t md_index,
-                                           const ucp_config_t *config,
-                                           const uct_tl_resource_desc_t *resource,
-                                           uint8_t rsc_flags, unsigned *num_resources_p,
-                                           uint64_t dev_cfg_masks[],
-                                           uint64_t *tl_cfg_mask)
+static void
+ucp_add_tl_resource_if_enabled(ucp_context_h context, ucp_tl_md_t *md,
+                               ucp_md_index_t md_index,
+                               const ucp_config_t *config,
+                               const uct_tl_resource_desc_t *resource,
+                               unsigned *num_resources_p,
+                               uint64_t dev_cfg_masks[], uint64_t *tl_cfg_mask)
 {
+    uint8_t rsc_flags = 0;
```

Comment:
IMO ucp_is_resource_enabled should initialize *rsc_flags to 0

Reply:
Done.